checkout/index.md

```diff
@@ -40,15 +40,21 @@ To start integrating Swedbank Pay Checkout, you need the following:
 * Agreement that includes Swedbank Pay Checkout
 * Obtained credentials (merchant Access Token) from Swedbank Pay through
   Swedbank Pay Admin. Please observe that Swedbank Pay Checkout encompass
-  both the **`consumer`** and **`paymentmenu`** scope.
+  both the **`consumer`** and **`paymentmenu`** scope
```

Comment:
Please add the period back to this sentence.

Reply:
Would you also like a punctuation mark at the end of the sentences at line 39 and 40?

Reply:
Added the punctuation marks to the document and starting a new PR for the request.